docs/sources/configure-client/grafana-agent/ebpf/setup-linux.md

```diff
@@ -42,51 +40,41 @@ uname -r
 
 Make sure you have a kernel version >= 4.9.
 
-## Install the Grafana Agent
-
-Follow the [installation instructions][agent-install] to download and install the Grafana Agent for your current Linux distribution.
+## Install Grafana Alloy
 
-Verify that the agent is correctly installed by running:
+Follow the [installation instructions](https://grafana.com/docs/alloy/<ALLOY_VERSION>/set-up/install/linux/) to download and install Alloy for your current Linux distribution.
 
-```shell
-grafana-agent-flow --version
-```
+## Configure Grafana Alloy
 
-## Configure the Grafana Agent
+To configure the Alloy eBPF profiler to profile local processes, you'll need to set the `targets_only` flag to `false` and add a default target in the `pyroscope.ebpf` component.
```

Comment:
`targets_only` is an undocumented flag and was used as a workaround before `discovery.process` was available.

now they need to use `discovery.process` https://grafana.com/docs/pyroscope/latest/configure-client/grafana-agent/ebpf/setup-linux/